tests/network/localnet/conftest.py

```
@@ -474,3 +479,96 @@ def localnet_ovs_bridge_jumbo_frame_client_and_server_vms(
+@pytest.fixture()
+def localnet_nad_ipv4_subnets() -> str:
+    return f"{random_ipv4_address(net_seed=0, host_address=0)}/24"
+
+
+@pytest.fixture()
+def nad_localnet_secondary_nic(
+    admin_client: DynamicClient,
+    vlan_id: int,
+    localnet_nad_ipv4_subnets: str,
+    namespace_localnet_1: Namespace,
+    nncp_localnet_on_secondary_node_nic: libnncp.NodeNetworkConfigurationPolicy,
```

Ruff (0.15.2)

[warning] Unused function argument: `nncp_localnet_on_secondary_node_nic`

(ARG001)